docs/developer-docs/latest/development/typescript.md

```diff
@@ -109,11 +109,11 @@ To start Strapi programmatically in a TypeScript project the Strapi instance req
 Strapi can be run programmatically by using the `strapi()` factory. Since the code of TypeScript projects is compiled in a specific directory, the parameter `distDir` should be passed to the factory to indicate where the compiled code should be read:
 
 ```js
-// path: ./src/plugins/<plugin-name>/server/index.js 
+// path: ./server.js 
 
 const strapi = require('@strapi/strapi');
 
-const app = await strapi({ distDir: './dist' });
+const app = strapi({ distDir: './dist' });
```

Comment:
Aren't you missing `app.start()` after this line to launch it?

Reply:
Yes. I added this line. Please let me know if the code looks OK now.

Reply:
Looks perfect 👌 Thank you, Shaun!